src/components/NamespaceDropdown.tsx

```diff
@@ -55,8 +58,25 @@ export class NamespaceDropdown extends React.PureComponent<NamespaceListType, {}
 
   componentDidMount() {
     this.props.refresh();
+    this.syncNamespacesURLParam();
   }
 
+  componentDidUpdate(prevProps: NamespaceListType) {
+    if (prevProps.activeNamespaces !== this.props.activeNamespaces) {
+      HistoryManager.setParam(URLParams.NAMESPACES, this.props.activeNamespaces.map(item => item.name).join(','));
+    }
+  }
+
+  syncNamespacesURLParam = () => {
+    const namespaces = (HistoryManager.getParam(URLParams.NAMESPACES) || '').split(',').filter(Boolean);
```

Comment:
what's `.filter(Boolean)`?

Reply:
it's a safe way of clean the array of empty values because I need to get diff with the activateNamespaces of the selector